*** 2017-06-18 20:55:06.003 idempiere Log (CLogFile) ***
20:55:06.002 Adempiere.startup: iDempiere(r) Release 3.1_2015-10-31 -Smart Suite ERP, CRM and SCM- (c) 1999-2013 iDempiere(r); Implementation: iDempiere - Supported by iDempiere community [17]
20:55:06.006 Ini.loadProperties: E:\mercurialidempiere/idempiereDEMO.properties #33 [17]
20:55:10.263 AdempiereActivator.installPackage: ni.hmil.override.codebase 1.0.0 was installed: 2017-06-18 10:07:05.297 [17]
20:55:11.863 DB.isBuildOK: Build DB=${env.ADEMPIERE_VERSION} 20080428-1232 [17]
20:55:11.864 DB.isBuildOK: Build Cl=3.1.0.qualifier [17]
20:55:14.843-----------> DB.isBuildOK: Error en la versión de la construcción 

El programa asume version de construcción 3.1.0.qualifier, pero la base de datos tiene una versión ${env.ADEMPIERE_VERSION} 20080428-1232.
Es posible que esto cause errores difíciles de corregir.
Por favor contacte al ADMINISTRADOR. [17]
20:55:19.137 WebEnv.initWeb: Servlet Context Init Parameters: iDempiere Root
ServerRoot=1 [17]
20:55:19.372-----------> EMail.setFrom: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
20:55:19.875-----------> EMail.addTo: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
20:55:19.875-----------> EMail.isValid: From is invalid=null [17]
20:55:19.875-----------> EMail.isValid: From is invalid=null [17]
20:55:19.876-----------> MClient.sendEMail: Could NOT Send Email: Server started: iDempiere (odbz:192.168.47.1) to you @ company.org: Invalid Data (System) [17]
20:55:19.876 WebEnv.initWeb: Servlet Init Parameter: idempiereMonitor [17]
20:55:19.876 AdempiereMonitor.init:  [17]
20:55:55.798-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
20:55:56.067-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://182.fwk1399961333:2/metainfo/zk/config.xml
with bundleresource://182.fwk1399961333:19/metainfo/zk/config.xml [17]
20:55:56.073-----------> ClassLocator.getDependentXMLResources: Replicate resource: zk
Overwrite bundleresource://182.fwk1399961333:11/metainfo/zk/config.xml
with bundleresource://182.fwk1399961333:28/metainfo/zk/config.xml [17]
20:55:56.079-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://182.fwk1399961333:12/metainfo/zk/config.xml
with bundleresource://182.fwk1399961333:29/metainfo/zk/config.xml [17]
20:55:56.112-----------> ClassLocator.getDependentXMLResources: Replicate resource: zul
Overwrite bundleresource://182.fwk1399961333:14/metainfo/zk/zk.xml
with bundleresource://182.fwk1399961333:31/metainfo/zk/zk.xml [17]
20:55:56.947-----------> LanguageDefinition.<init>: Replicated language: xhtml, overriden by [LanguageDefinition: xhtml] [17]
20:55:56.981-----------> LanguageDefinition.<init>: Replicated language: xul/html, overriden by [LanguageDefinition: xul/html] [17]
20:55:57.044-----------> ClassLocator.getDependentXMLResources: Replicate resource: calendar
Overwrite bundleresource://182.fwk1399961333:1/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1399961333:18/metainfo/zk/lang-addon.xml [17]
20:55:57.048-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://182.fwk1399961333:2/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1399961333:19/metainfo/zk/lang-addon.xml [17]
20:55:57.058-----------> ClassLocator.getDependentXMLResources: Replicate resource: gmapsz
Overwrite bundleresource://182.fwk1399961333:5/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1399961333:22/metainfo/zk/lang-addon.xml [17]
20:55:57.062-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeline
Overwrite bundleresource://182.fwk1399961333:6/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1399961333:23/metainfo/zk/lang-addon.xml [17]
20:55:57.065-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeplot
Overwrite bundleresource://182.fwk1399961333:7/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1399961333:24/metainfo/zk/lang-addon.xml [17]
20:55:57.082-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://182.fwk1399961333:12/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1399961333:29/metainfo/zk/lang-addon.xml [17]
20:55:57.092-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkplus
Overwrite bundleresource://182.fwk1399961333:13/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1399961333:30/metainfo/zk/lang-addon.xml [17]
20:55:57.840           WebUIServlet.init: iDempiere Web Client started successfully [17]
20:55:59.080-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
20:55:59.203-----------> LCO_ValidatorWH.initialize:  [48]

*** 2017-06-18 20:55:59.373 idempiere Log (CLogFile) ***
20:56:02.264-----------> Node: [192.168.163.1]:5701 [adempiere.idempiere/localhost] [3.5.3] No join method is enabled! Starting standalone. [50]
21:00:10.504===========> TableIndexElementHandler.validateTableIndex: ERROR: column am.amcanorder does not exist
  Position: 427 [41]
org.postgresql.util.PSQLException: ERROR: column am.amcanorder does not exist
  Position: 427; State=42703; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
	at org.postgresql.jdbc2.AbstractJdbc2DatabaseMetaData.getIndexInfo(AbstractJdbc2DatabaseMetaData.java:4230)
	at com.mchange.v2.c3p0.impl.NewProxyDatabaseMetaData.getIndexInfo(NewProxyDatabaseMetaData.java:3328)
	at org.compiere.process.TableIndexValidate.validateTableIndex(TableIndexValidate.java:69)
	at org.adempiere.pipo2.handler.TableIndexElementHandler.validateTableIndex(TableIndexElementHandler.java:112)
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:93)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)

21:00:18.070===========> PackIn.importXML: importXML: [41]
org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

21:00:21.711===========> PackIn.importXML: importXML: [41]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

21:00:24.614===========> PipoDictionaryService.merge: importXML: [41]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

21:00:37.905===========> Incremental2PackActivator.packIn: Pack in failed. [41]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

21:15:36.311-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks! [43]
21:15:36.317-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@13c43965
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@6d71965
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@636cf72a
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@37942006
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@5dc2866b
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@3c30df7f
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@2ca129b8
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@3f84ecb6
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@2fd0c14e
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@d2d757
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@1237912d
Pool thread stack traces:
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
 [43]
21:35:36.375-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks! [43]
21:35:36.381-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@1eca6c1b
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@ffb1ad4
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@2e9e5c7e
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@6bacb988
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@18c8c6f2
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@5a6401d6
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@5bb4b6f6
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@5d5e288a
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@50b7abe9
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@791af9f2
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@5a0fbcfb
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@3ca92972
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@5e01a7b
Pool thread stack traces:
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
 [43]
21:55:36.434-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks! [43]
21:55:36.440-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@10e2e6db
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@20a23c86
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@76c03bfa
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@54c69a01
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@25d7eb44
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@27ed8d28
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@5cce5192
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@28642834
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@478b82a3
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@25b936bf
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@22b948f2
Pool thread stack traces:
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
 [43]
22:05:50.423===========> PreparedStatementProxy.init: SELECT DISTINCT ProcessedOn FROM GL_Journal WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [59]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:05:55.002===========> AcctProcessor.postSession: SELECT DISTINCT ProcessedOn FROM GL_Journal WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [59]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:06:12.008===========> PreparedStatementProxy.init: SELECT * FROM AD_Column WHERE AD_Table_ID=? AND IsActive='Y' ORDER BY ColumnName [135]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.model.MTable.getColumns(MTable.java:247)
	at org.compiere.model.MTable.getColumn(MTable.java:281)
	at org.compiere.model.MSequence.getNextID(MSequence.java:191)
	at org.compiere.util.DB.getNextID(DB.java:1840)
	at org.compiere.model.PO.saveNew(PO.java:2711)
	at org.compiere.model.PO.save(PO.java:2128)
	at org.compiere.model.PO.saveEx(PO.java:2216)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:88)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:06:14.836===========> MTable.getColumns: SELECT * FROM AD_Column WHERE AD_Table_ID=? AND IsActive='Y' ORDER BY ColumnName [135]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.model.MTable.getColumns(MTable.java:247)
	at org.compiere.model.MTable.getColumn(MTable.java:281)
	at org.compiere.model.MSequence.getNextID(MSequence.java:191)
	at org.compiere.util.DB.getNextID(DB.java:1840)
	at org.compiere.model.PO.saveNew(PO.java:2711)
	at org.compiere.model.PO.save(PO.java:2128)
	at org.compiere.model.PO.saveEx(PO.java:2216)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:88)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.model.MTable.getColumns(MTable.java:247)
	at org.compiere.model.MTable.getColumn(MTable.java:281)
	at org.compiere.model.MSequence.getNextID(MSequence.java:191)
	at org.compiere.util.DB.getNextID(DB.java:1840)
	at org.compiere.model.PO.saveNew(PO.java:2711)
	at org.compiere.model.PO.save(PO.java:2128)
	at org.compiere.model.PO.saveEx(PO.java:2216)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:88)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:06:15.957===========> PreparedStatementProxy.init: SELECT COUNT(1) FROM RV_Unprocessed WHERE AD_Client_ID=? AND CreatedBy=? [32]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)

22:06:19.875===========> DB.getSQLValue: SELECT COUNT(1) FROM RV_Unprocessed WHERE AD_Client_ID=? AND CreatedBy=? [32]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)
	at org.adempiere.webui.dashboard.DPActivities.refresh(DPActivities.java:129)
	at org.adempiere.webui.dashboard.DashboardRunnable.refreshDashboard(DashboardRunnable.java:131)
	at org.adempiere.webui.dashboard.DashboardRunnable.run(DashboardRunnable.java:92)
	at org.adempiere.webui.desktop.DashboardController$3.onEvent(DashboardController.java:461)
	at org.zkoss.zk.ui.AbstractComponent.onEvent(AbstractComponent.java:2773)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2744)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2685)
	at org.zkoss.zk.ui.impl.EventProcessor.process(EventProcessor.java:136)
	at org.zkoss.zk.ui.impl.UiEngineImpl.processEvent(UiEngineImpl.java:1765)
	at org.zkoss.zk.ui.impl.UiEngineImpl.process(UiEngineImpl.java:1550)
	at org.zkoss.zk.ui.impl.UiEngineImpl.execUpdate(UiEngineImpl.java:1260)
	at org.zkoss.zk.au.http.DHtmlUpdateServlet.process(DHtmlUpdateServlet.java:611)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)

22:06:31.459===========> PreparedStatementProxy.init: SELECT * FROM M_MatchPO WHERE AD_Client_ID=? AND (ProcessedOn IS NULL OR ProcessedOn=0) AND Processed='Y' AND Posted='N' AND IsActive='Y' ORDER BY Created [59]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:06:34.427===========> AcctProcessor.postSession: SELECT * FROM M_MatchPO WHERE AD_Client_ID=? AND (ProcessedOn IS NULL OR ProcessedOn=0) AND Processed='Y' AND Posted='N' AND IsActive='Y' ORDER BY Created [59]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:27:17.044===========> PreparedStatementProxy.init: SELECT DISTINCT ProcessedOn FROM A_Asset_Reval WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [71]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:27:36.522===========> AcctProcessor.postSession: SELECT DISTINCT ProcessedOn FROM A_Asset_Reval WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [71]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:48:37.653===========> PreparedStatementProxy.init: SELECT * FROM M_MatchPO WHERE AD_Client_ID=? AND (ProcessedOn IS NULL OR ProcessedOn=0) AND Processed='Y' AND Posted='N' AND IsActive='Y' ORDER BY Created [129]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:48:41.356===========> AcctProcessor.postSession: SELECT * FROM M_MatchPO WHERE AD_Client_ID=? AND (ProcessedOn IS NULL OR ProcessedOn=0) AND Processed='Y' AND Posted='N' AND IsActive='Y' ORDER BY Created [129]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:48:57.478===========> PreparedStatementProxy.init: SELECT * FROM A_Asset_Addition WHERE AD_Client_ID=? AND (ProcessedOn IS NULL OR ProcessedOn=0) AND Processed='Y' AND Posted='N' AND IsActive='Y' ORDER BY Created [128]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:49:01.088===========> AcctProcessor.postSession: SELECT * FROM A_Asset_Addition WHERE AD_Client_ID=? AND (ProcessedOn IS NULL OR ProcessedOn=0) AND Processed='Y' AND Posted='N' AND IsActive='Y' ORDER BY Created [128]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:183)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

22:55:16.602-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks! [43]
22:55:16.606-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@686aef7c
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@651ac89
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@4490eb72
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@5c16599c
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@54c62ad7
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@1e26b4bc
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@7f23b21a
Pool thread stack traces:
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
 [43]
22:55:17.629===========> PreparedStatementProxy.init: SELECT COUNT(1) FROM RV_Unprocessed WHERE AD_Client_ID=? AND CreatedBy=? [127]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)

22:55:24.849===========> DB.getSQLValue: SELECT COUNT(1) FROM RV_Unprocessed WHERE AD_Client_ID=? AND CreatedBy=? [127]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)
	at org.adempiere.webui.dashboard.DPActivities.refresh(DPActivities.java:129)
	at org.adempiere.webui.dashboard.DashboardRunnable.refreshDashboard(DashboardRunnable.java:131)
	at org.adempiere.webui.dashboard.DashboardRunnable.run(DashboardRunnable.java:92)
	at org.adempiere.webui.desktop.DashboardController$3.onEvent(DashboardController.java:461)
	at org.zkoss.zk.ui.AbstractComponent.onEvent(AbstractComponent.java:2773)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2744)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2685)
	at org.zkoss.zk.ui.impl.EventProcessor.process(EventProcessor.java:136)
	at org.zkoss.zk.ui.impl.UiEngineImpl.processEvent(UiEngineImpl.java:1765)
	at org.zkoss.zk.ui.impl.UiEngineImpl.process(UiEngineImpl.java:1550)
	at org.zkoss.zk.ui.impl.UiEngineImpl.execUpdate(UiEngineImpl.java:1260)
	at org.zkoss.zk.au.http.DHtmlUpdateServlet.process(DHtmlUpdateServlet.java:611)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)

23:08:18.177===========> PreparedStatementProxy.init: SELECT COUNT(1) FROM RV_Unprocessed WHERE AD_Client_ID=? AND CreatedBy=? [89]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)

23:08:21.586===========> DB.getSQLValue: SELECT COUNT(1) FROM RV_Unprocessed WHERE AD_Client_ID=? AND CreatedBy=? [89]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)
	at org.adempiere.webui.dashboard.DPActivities.refresh(DPActivities.java:129)
	at org.adempiere.webui.dashboard.DashboardRunnable.refreshDashboard(DashboardRunnable.java:131)
	at org.adempiere.webui.dashboard.DashboardRunnable.run(DashboardRunnable.java:92)
	at org.adempiere.webui.desktop.DashboardController$3.onEvent(DashboardController.java:461)
	at org.zkoss.zk.ui.AbstractComponent.onEvent(AbstractComponent.java:2773)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2744)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2685)
	at org.zkoss.zk.ui.impl.EventProcessor.process(EventProcessor.java:136)
	at org.zkoss.zk.ui.impl.UiEngineImpl.processEvent(UiEngineImpl.java:1765)
	at org.zkoss.zk.ui.impl.UiEngineImpl.process(UiEngineImpl.java:1550)
	at org.zkoss.zk.ui.impl.UiEngineImpl.execUpdate(UiEngineImpl.java:1260)
	at org.zkoss.zk.au.http.DHtmlUpdateServlet.process(DHtmlUpdateServlet.java:611)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getUnprocessedCount(DPActivitiesModel.java:141)

23:09:53.352===========> PreparedStatementProxy.init: SELECT DISTINCT ProcessedOn FROM C_AllocationHdr WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [135]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:09:56.030===========> AcctProcessor.postSession: SELECT DISTINCT ProcessedOn FROM C_AllocationHdr WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [135]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:10:19.509===========> PreparedStatementProxy.init: SELECT DISTINCT ProcessedOn FROM A_Depreciation_Entry WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [135]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:10:22.224===========> AcctProcessor.postSession: SELECT DISTINCT ProcessedOn FROM A_Depreciation_Entry WHERE AD_Client_ID=? AND ProcessedOn<? AND Processed='Y' AND Posted='N' AND IsActive='Y' [135]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:127)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:30:40.882===========> PreparedStatementProxy.init: SELECT statement_timestamp() [197]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueTSEx(DB.java:1521)
	at org.compiere.util.DB.getSQLValueTS(DB.java:1565)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:100)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:31:00.090===========> DB.getSQLValueTS: SELECT CURRENT_TIMESTAMP FROM DUAL [197]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueTSEx(DB.java:1521)
	at org.compiere.util.DB.getSQLValueTS(DB.java:1565)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:100)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueTSEx(DB.java:1521)
	at org.compiere.util.DB.getSQLValueTS(DB.java:1565)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:100)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:41:46.627===========> PreparedStatementProxy.init: SELECT statement_timestamp() [200]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueTSEx(DB.java:1521)
	at org.compiere.util.DB.getSQLValueTS(DB.java:1565)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:100)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:42:05.838===========> DB.getSQLValueTS: SELECT CURRENT_TIMESTAMP FROM DUAL [200]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueTSEx(DB.java:1521)
	at org.compiere.util.DB.getSQLValueTS(DB.java:1565)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:100)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueTSEx(DB.java:1521)
	at org.compiere.util.DB.getSQLValueTS(DB.java:1565)
	at org.compiere.server.AcctProcessor.postSession(AcctProcessor.java:100)
	at org.compiere.server.AcctProcessor.doWork(AcctProcessor.java:79)
	at org.compiere.server.AdempiereServer.doRun(AdempiereServer.java:211)
	at org.compiere.server.AdempiereServer.run(AdempiereServer.java:190)
	at org.compiere.server.AdempiereServerMgr$ServerWrapper.run(AdempiereServerMgr.java:504)

23:55:16.761-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks! [43]
23:55:16.766-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@38ce3b00 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@1ba86338
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@5340aab
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@4c09ffb7
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@24736e61
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@1653b4f9
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@4963c124
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@341d4820
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@6e6a479a
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@3830818
Pool thread stack traces:
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#2,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#1,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1gslsi31pk4gv9|149717ec, dataSourceName->iDempiereDS]-HelperThread-#0,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
 [43]
23:56:16.782-----------> ThreadPoolAsynchronousRunner.log: Task com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@4c09ffb7 (in deadlocked PoolThread) failed to complete in maximum time 60000ms. Trying interrupt(). [43]

*** 2017-06-19 00:01:52.963 idempiere Log (CLogFile) ***
